dauthz/backends/casbin_backend.py

```diff
@@ -0,0 +1,87 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.backends import BaseBackend
+
+from dauthz.core import enforcer, enforcers
+
+
+UserModel = get_user_model()
+
+
+class CasbinBackend(BaseBackend):
+    """
+    Check permissions with Casbin.
+    """
+
+    def __init__(self):
+        self.enforcer = enforcer
+
+    def authenticate(self, request, username=None, password=None, **kwargs):
+        if username is None:
+            username = kwargs.get(UserModel.USERNAME_FIELD)
+        if username is None or password is None:
+            return
+        try:
+            user = UserModel._default_manager.get_by_natural_key(username)
+        except UserModel.DoesNotExist:
+            # Run the default password hasher once to reduce the timing
+            # difference between an existing and a nonexistent user (#20760).
+            UserModel().set_password(password)
+        else:
+            if user.check_password(password) and self.user_can_authenticate(user):
+                return user
+
```

Comment:
Can user authentication be skipped? Hand it over to other Backends to complete.
Only the authorization of permissions is provided here.

Reply:
Django will use each backend in order, if users want to custom their own authentication function, they can define a new backend and place it before casbin_backend.

Reply:
I see. I will file a new pr to remove it.